app/scss/_style.scss

```diff
@@ -811,6 +811,14 @@ img {
     position: absolute;
     z-index: 2;
   }
+
+  .item-xp-bar-small {
+    position: absolute;
+    top: 0px;
+    height: 2px;
+    opacity: .95;
```

Comment:
More specifically, why the opacity? It's always going to be against a white background.